### PR TITLE
Aemceds 66 org page

### PR DESCRIPTION
**Pull request by akki761**
Fixes # [AEMCEDS-66](https://jira.corp.adobe.com/browse/AEMCEDS-66)

Test URLs:
- Before: [https://main--aem-competency--aem-comp.hlx.live/drafts/Akash/core-team](https://main--aem-competency--aem-comp.hlx.live/drafts/Akash/core-team)
- After: [https://aemceds-66-org-page--aem-competency--aem-comp.hlx.live/drafts/Akash/core-team](https://aemceds-66-org-page--aem-competency--aem-comp.hlx.live/drafts/Akash/core-team)

### CODE REVIEW

File: blocks/columns/columns.js [hunk 1]
```diff
@@ -13,6 +15,12 @@ export default function decorate(block) {
           picWrapper.classList.add('columns-img-col');
         }
       }
+      if (block.classList.contains('core-team') && !col.classList.contains('columns-img-col')) {
```
Comment (mkbansal1) [inline blocks/columns/columns.js]: Statement "block.classList.contains('core-team')" is executed in loop, suggest to put it in an object outside loop and use object.

File: styles/styles.css [hunk 2]
```diff
@@ -100,6 +100,8 @@
   --button-tertiary-border-color: var(--button-tertiary-text-color); 
   --button-tertiary-border-hover-color: var(--button-tertiary-bg-hover-color); 
   --header-shadow-color: #00000029;
+  --org-card-desc-bg-color: #0008;
```
Comment (mkbansal1) [inline styles/styles.css]: try using generic variable name.

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
- Change variation name, make it generic like "team-view"

File: blocks/columns/columns.js (hunk 1 above)
Comment (akki761) [reply blocks/columns/columns.js]: @mkbansal1 I will remove this JS. This is not required.

File: styles/styles.css (hunk 2 above)
Comment (akki761) [reply styles/styles.css]: will do

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**